velox/docs/functions/spark/datetime.rst

```diff
@@ -213,6 +213,35 @@ These functions support TIMESTAMP and DATE input types.
 
         SELECT month('2009-07-30'); -- 7
 
+.. spark:function:: months_between(timestamp1, timestamp2) -> double
+
+    	If `timestamp1` is later than `timestamp2`, then the result is positive.
```

Comment:
Perhaps add a description for the function's purpose.
> Returns number of months between times \`\`timestamp1\`\` and \`\`timestamp2\`\`.

Reply:
Done